openpype/hosts/webpublisher/webserver_service/webpublish_routes.py

```diff
@@ -263,11 +266,14 @@ async def post(self, request) -> Response:
                     args.append(value)
 
         log.info("args:: {}".format(args))
+        if content.get("studio_processing"):
```

Comment:
Could that be done with some key in `studio_processing_filters` as this will break TVPaint processing?

Reply:
Ok, updated it.